src/errors.lisp

```diff
@@ -1,5 +1,10 @@
 (in-package #:sbcl-librarian)
 
+(define-type :json
+  :c-type "char*"
+  :python-type "c_char_p"
+  :alien-type sb-alien:c-string)
```

Comment:
Does anything else in this MR consult this? (Also, might be nice if exhaust-heap were elsewhere: maybe later)

Reply:
No, but it is used downstream